### PR TITLE
4435 - Metadata upload needs all works

### PR DESCRIPTION
**Pull request by WillNigel23**

### CODE REVIEW

**Comment by WillNigel23** [comment]
Please review translations :pray:

File: config/locales/metadata/metadata-fr.yml [hunk 1]
```diff
@@ -17,4 +17,4 @@ fr:
       template_file_anchor: ce fichier modèle
       upload: Télécharger
       upload_metadata: Télécharger les métadonnées
-      upload_metadata_description: Pour mettre à jour les métadonnées de plusieurs œuvres de cette collection, téléchargez une feuille de calcul avec une ligne par œuvre. Veuillez télécharger %{link} pour commencer.
+      upload_metadata_description: Pour mettre à jour les métadonnées de plusieurs œuvres de cette collection, téléchargez une feuille de calcul avec une ligne par œuvre. <br> Les colonnes avec <i>étoiles</i> ne peuvent pas être modifiées. Ignorez ou supprimez toutes les colonnes que vous ne modifiez pas. <br> Veuillez télécharger %{link} pour commencer.
```
Comment (benwbrum) [inline config/locales/metadata/metadata-fr.yml:20]: Change `étoiles` to `astérisques`

File: config/locales/metadata/metadata-de.yml [hunk 2]
```diff
@@ -17,4 +17,4 @@ de:
       template_file_anchor: diese Vorlage
       upload: Hochladen
       upload_metadata: Metadaten hochladen
-      upload_metadata_description: Um Metadaten für mehrere Werke in dieser Sammlung zu aktualisieren, laden Sie eine Tabelle mit einer Zeile pro Werk hoch. Bitte laden Sie %{link} herunter, um zu beginnen.
+      upload_metadata_description: Um Metadaten für mehrere Werke in dieser Sammlung zu aktualisieren, laden Sie eine Tabelle mit einer Zeile pro Werk hoch. <br> Spalten mit <i>Sternen</i> können nicht geändert werden. Ignorieren oder löschen Sie alle Spalten, die Sie nicht ändern. <br> Bitte laden Sie %{link} herunter, um zu beginnen.
```
Comment (benwbrum) [inline config/locales/metadata/metadata-de.yml:20]: Change `Sternen` to `Sternchen`

File: config/locales/metadata/metadata-es.yml [hunk 3]
```diff
@@ -17,4 +17,4 @@ es:
       template_file_anchor: este archivo de plantilla
       upload: Subir
       upload_metadata: Subir metadatos
-      upload_metadata_description: Para actualizar los metadatos de varias obras dentro de esta colección, cargue una hoja de cálculo con una fila por obra. Descargue %{link} para comenzar.
+      upload_metadata_description: Para actualizar los metadatos de varias obras dentro de esta colección, cargue una hoja de cálculo con una fila por obra. <br> Las columnas con <i>estrellas</i> no se pueden modificar. Ignore o elimine cualquier columna que no esté cambiando. <br> Descargue %{link} para comenzar.
```
Comment (benwbrum) [inline config/locales/metadata/metadata-es.yml:20]: This is fine: "Los ingenieros del software y los científicos lo denominan a veces como «estrella»", according to Wikipedia.

File: config/locales/metadata/metadata-pt.yml [hunk 4]
```diff
@@ -17,4 +17,4 @@ pt:
       template_file_anchor: este arquivo de modelo
       upload: Envio
       upload_metadata: Carregar metadados
-      upload_metadata_description: Para atualizar os metadados de várias obras desta coleção, carregue uma planilha com uma linha por obra. Faça o download do %{link} para começar.
+      upload_metadata_description: Para atualizar os metadados de várias obras desta coleção, carregue uma planilha com uma linha por obra. <br> Faça o download do %{link} para começar.
```
Comment (benwbrum) [inline config/locales/metadata/metadata-pt.yml:20]: This does not seem to contain the note about stars.

**Review by benwbrum — CHANGES_REQUESTED** [review]
A few translation changes.

**Review by saracarl — CHANGES_REQUESTED** [review]
This can be merged once the translations are fixed.

File: config/locales/metadata/metadata-es.yml (hunk 3 above)
Comment (WillNigel23) [reply config/locales/metadata/metadata-es.yml:20]: Sorry, can you please clarify what we want here instead?

File: config/locales/metadata/metadata-pt.yml (hunk 4 above)
Comment (WillNigel23) [reply config/locales/metadata/metadata-pt.yml:20]: I believe I used the google_translate api you provided here. Can you please let me know the correct translation here? (Given that the translate api key seems to have translated this wrong?)

File: config/locales/metadata/metadata-es.yml (hunk 3 above)
Comment (benwbrum) [reply config/locales/metadata/metadata-es.yml:20]: Sorry -- I meant that _estrella_ was fine in a technical context.  (I'd been trying to make sure we didn't need to translate is as _asterisco_.)

File: config/locales/metadata/metadata-pt.yml (hunk 4 above)
Comment (benwbrum) [reply config/locales/metadata/metadata-pt.yml:20]: Strange -- this looks fine now.  My apologies.

**Pull request merged**